tests/lib/fakestore/store/store.go

```diff
@@ -98,7 +98,7 @@ func NewStore(topDir, addr string, assertFallback bool) *Store {
 	mux.HandleFunc("/api/v1/snaps/details/", store.detailsEndpoint)
 	mux.HandleFunc("/api/v1/snaps/metadata", store.bulkEndpoint)
 	mux.Handle("/download/", http.StripPrefix("/download/", http.FileServer(http.Dir(topDir))))
-	mux.HandleFunc("/assertions/", store.assertionsEndpoint)
+	mux.HandleFunc("/api/v1/snaps/assertions/", store.assertionsEndpoint)
```

Comment:
It's true that we could continue setting `SNAPPY_FORCE_SAS_URL` when starting the fakestore, but I prefer it this way: it means that we're closer to faking a single upstream URL namespace rather than blatting several URL namespaces from different domains together (though not entirely there yet).

Do you have a specific reason to prefer not to make this change?